core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -345,13 +346,13 @@ describe('Creator dashboard controller', () => {
         expect(ctrl.relativeChangeInTotalPlays).toBe(5);
       });
 
-      it('should change active tab name', function() {
+      it('should change active tab name when clicking on tab', function() {
         expect(ctrl.activeTab).toBe('myExplorations');
         ctrl.setActiveTab('suggestions');
         expect(ctrl.activeTab).toBe('suggestions');
       });
 
-      it('should set my explorations view correctly', function() {
+      it('should set explorations to backend when chaning view', function() {
```

Comment:
"chaning" is misspelled. Even with that fix, though, I don't understand this description.

Reply:
Done.